ecoscope/analysis/UD/etd_range.py

```diff
@@ -83,7 +83,7 @@ class Weibull3Parameter(WeibullPDF):
 
 
 def calculate_etd_range(
-    trajectory_gdf: Trajectory,
+    trajectory_gdf: gpd.GeoDataFrame,
```

Comment:
Love that this is now possible.

Do we want to be more opinionated about typing here, perhaps via pandera schemas?

Certainly the majority of `gpd.GeoDataFrame`s will not work if passed here.

Not necessarily a question to be solved by this PR, but something to consider as a follow-on perhaps.